magento/magento2
# Aws s3

Using the CLI command and passing `--remote-storage-path-style=1` should process the option.
The config is then converted to `remote_storage/config/path-style` but in the factory `path_style` is beeing read.

### Description (*)
This pull request fixes the typo of the configuration option.

### Manual testing scenarios (*)
1. Using a existing magento installation
2. Installing Minio https://min.io/ on a remote server
3. Run the command `bin/magento setup:config:set --remote-storage-endpoint="xxxxxxxxxxxx" --remote-storage-driver="aws-s3"  --remote-storage-bucket="xxxxxxxxxxxx"  --remote-storage-region="xxxxxxxxxxxx" --remote-storage-key="xxxxxxxxxxxx" --remote-storage-secret="xxxxxxxxxxxx" --remote-storage-path-style="1"`
4. See that `--remote-storage-path-style` is allowed but ignored and `--remote-storage-path_style` is not allowed


### Contribution checklist (*)
